src/Operation/Explainable.php

```diff
@@ -27,8 +27,6 @@ interface Explainable extends Executable
 {
     /**
      * Returns the command document for this operation.
-     *
-     * @return array
      */
-    public function getCommandDocument();
+    public function getCommandDocument(): array;
```

Comment:
We currently return arrays for all commands - this major version would be a chance to create objects instead if we wanted to. I don't have any strong feelings about this, but wanted to mention it.

Reply:
The pedantic choice would probably be `array|object` for consistency with [`MongoDB\Driver\Command`](https://www.php.net/manual/en/class.mongodb-driver-command.php).

**/me glances at the wall**

[Covariance](https://www.php.net/manual/en/language.oop5.variance.php) would still allow us to maintain `array` type hints in the operation class implementations: https://3v4l.org/AvFKa

And I don't think the Explain operation would require any changes one way or another.